packages/wonder-blocks-tooltip/components/tooltip-content.js

```diff
@@ -1,9 +1,11 @@
 // @flow
 import * as React from "react";
+
 import {View} from "@khanacademy/wonder-blocks-core";
 import {Strut} from "@khanacademy/wonder-blocks-layout";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {HeadingSmall, LabelMedium} from "@khanacademy/wonder-blocks-typography";
+
 import type {Typography} from "@khanacademy/wonder-blocks-typography";
```

Comment:
Thanks for following webapp style.  I haven't been raising this as an issue in reviews, maybe I should be.